zsl/service/service.py

```diff
@@ -43,21 +45,20 @@ def __init__(self, app, engine):
 
 def transactional(f):
     @wraps(f)
-    @inject(app=Zsl)
-    def transactional_f(app=Injected, *args, **kwargs):
+    def transactional_f(*args, **kwargs):
```

Comment:
Funny thing the `@inject(app=Zsl)` was failing on some exception, which I could not fix, but then I realized it was needed only for the logger, which we started to change to `logging.blabla` anyway.